src/_provisional.scss

```diff
@@ -417,6 +417,25 @@ $provisional-alert-success-color: #278400;
 			}
 		}
 	}
+
+	&.from {
```

Comment:
Change this selector as discussed earlier in another file

Reply:
Changed css selector